.github/workflows/kaito-e2e.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
           echo "VERSION=${rand}" >> $GITHUB_ENV
           echo "CLUSTER_NAME=kaito${rand}" >> $GITHUB_ENV
           echo "RUN_LLAMA_13B=false" >> $GITHUB_ENV
-          echo "AI_MODELS_IMAGE_VERSION=0.0.1" >> $GITHUB_ENV 
+          echo "AI_MODELS_IMAGE_VERSION=0.0.3" >> $GITHUB_ENV 
```

Comment:
PR: https://github.com/Azure/kaito/pull/252 will take care of setting this based on supported_models.yaml